source/BoardingPanel.cpp

```diff
@@ -504,6 +490,37 @@ bool BoardingPanel::CanAttack() const
 }
 
 
+// Handle the keyboard scrolling and selection in the panel list.
+void BoardingPanel::DoKeyboardNavigation(const SDL_Keycode key)
+{
+		// Scrolling the list of plunder.
+		if(key == SDLK_PAGEUP || key == SDLK_PAGEDOWN)
+			// With 220 px of content and 20 px height per line, there are 11
+			// lines. However, although some programs scroll a whole page
+			// (e.g. most KDE programs) it seems to be more common to scroll a
+			// bit less than a whole page (e.g. the most used browsers or
+			// common IDEs), so scroll only 10 lines.
```

Comment:
This comment is a bit much. It's fine to just say e.g. "Keep one of the previous items onscreen while paging through."

Reply:
I changed it as suggested.